src/containers/access/src/reducers/getAccessRequestReducer.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { ReduxConstants } from 'lattice-utils';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import {
+  GET_ACCESS_REQUEST,
+  getAccessRequest,
+} from '../actions';
+
+const { REQUEST_STATE } = ReduxConstants;
+
+export default function getAccessRequestReducer(state :Map<*, *>, action :SequenceAction) {
+  return getAccessRequest.reducer(state, action, {
+    REQUEST: () => state.setIn([GET_ACCESS_REQUEST, REQUEST_STATE], RequestStates.PENDING),
+    SUCCESS: () => state
+      .merge(action.value)
+      .setIn([GET_ACCESS_REQUEST, REQUEST_STATE], RequestStates.SUCCESS),
+    FAILURE: () => state.setIn([GET_ACCESS_REQUEST, REQUEST_STATE], RequestStates.FAILURE),
+  });
```

Comment:
how about adding a FINALLY case here to delete the action.id since the saga calls `getAccessRequest.finally`?

Reply:
the id is not stored explicitly. getAccessRequest.finally already internally clears the id